test/utils.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const t = require('tap')
+const { convertUrlToWebSocket } = require('../utils')
+
+t.test('convertUrlToWebSocket', function (t) {
```

Comment:
tap has a built-in way to create array of tests, check it out. also, before and after are not two very meaningful names for inputs and outputs of the tests. more idiomatic names are e.g. input/expected

Reply:
turns out I was wong!